view-generator-framework/src/main/java/org/hypertrace/core/viewgenerator/service/ViewGeneratorLauncher.java

```diff
@@ -53,7 +54,22 @@ public StreamsBuilder buildTopology(Map<String, Object> properties, StreamsBuild
       inputStream = streamsBuilder.stream(inputTopic, Consumed.with(Serdes.String(), null));
       inputStreams.put(inputTopic, inputStream);
     }
-    inputStream.flatMapValues(viewMapper).to(outputTopic, Produced.with(Serdes.String(), null));
+
+    // This enviorn property helps in overriding producer value serde.
+    // In local mode of deployment of single ingestion pipeline, this helps
+    // in using GenericAvroSerde which is independent of schema registry.
+    Serde producerValueSerde = null;
+    String envProducerValueSerdeClassName = System.getProperty("PRODUCER_VALUE_SERDE");
```

Comment:
System.getProperty is not an environment property ?

Reply:
Addressed.